proteus/mprans/RANS3PF2D.h

```diff
@@ -4621,11 +4641,12 @@ namespace proteus
                     int ebN = elementBoundariesArray[eN*nDOF_mesh_trial_element+opp_node];//only works for simplices
                     surrogate_boundaries.push_back(ebN);
                     //now find which element neighbor this element is
-                    if (eN == elementBoundaryElementsArray[eN*2+0])
-                      surrogate_boundary_elements.push_back(1);
+                    if (eN == elementBoundaryElementsArray[ebN*2+0])
```

Comment:
the old code and the new code is equivalent here, right? You're just asserting what the old code was assuming?